intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -522,12 +830,12 @@ \subsection{Get detailed info}
 -------------------------------------------------------------------
 /apps/\site/\hpcname/modules/all/scripts/2.6.0:
```

Comment:
don't use the `scripts` module as an example, since it's a special case

use `GCC` instead?